packages/create-expo-module/src/prompts.ts

```diff
@@ -0,0 +1,77 @@
+import path from 'path';
+import { Answers, PromptObject } from 'prompts';
+import validateNpmPackage from 'validate-npm-package-name';
+
+import { findGitHubEmail, findGitHubProfileUrl, findMyName, guessRepoUrl } from './utils';
+
+export default async function getPrompts(targetDir: string): Promise<PromptObject<string>[]> {
+  const targetBasename = path.basename(targetDir);
+
+  return [
+    {
+      type: 'text',
+      name: 'slug',
+      message: 'What is the name of the npm package?',
+      initial: validateNpmPackage(targetBasename).validForNewPackages ? targetBasename : undefined,
+      validate: (input) =>
+        validateNpmPackage(input).validForNewPackages || 'Must be a valid npm package name',
+    },
+    {
+      type: 'text',
+      name: 'name',
+      message: 'What is the project name?',
```

Comment:
it's unclear what we mean by "project name" here. should we maybe say what we use this for? or can you think of something else we can say that would be more clear?

Reply:
This is the name for iOS pod, but also name of the native module under which it's exported to JS. I hope it would be more clear to say it's the native module name 🤔
By the way, I'd like to modify `expo prebuild` so we could customize the name of iOS project of the example app to make it `MyExpoModuleExample` instead of lowercased `myexpomoduleexample` 😅